src/containers/ErrorPage/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
+import Helmet from 'react-helmet';
+import { StatusAlert } from '@edx/paragon';
+
+import H1 from '../../components/H1';
+
+const ErrorPage = (props) => {
+  const { location } = props;
+  if (location.state && location.state.error) {
+    const { error } = location.state;
+
+    return (
+      <div className="container">
+        {error.status === 404 ? (
+          <Redirect to="/404" />
```

Comment:
I don't love the double redirect here, first to /error and then to /404 but it is probably okay for now...